docs/build/local-first.md

```diff
@@ -1,25 +1,31 @@
 ---
-sidebar_label: Local-first cache
+sidebar_label: Local-first architecture
 sidebar_position: 3
 description: "If you're building a production-grade app, be sure to use a local-first architecture to help you build a performant app."
 ---
 
 # Use local-first architecture
 
-If you're building a production-grade app, be sure to use a local-first architecture to help you build a performant app. Using this local-first architecture, the client prioritizes using the local cache on the device where it’s running.
+If you're building a production-grade app, be sure to use a local-first architecture to help you build a performant app. Using a local-first architecture means building your app so a client prioritizes using local storage on the device where it’s running.
```

Comment:
At this overview level, I have genericized "local cache" to "local storage." 

Deeper into the doc, we then start referring to "local cache" and "local database" as two types of local storage. I believe that local cache and local database are similar - but distinct concepts that we want to keep separate, correct?

Reply:
not sure if it's necessary to make the distinction here unless you want to explicitly differentiate the built-in V3 local DB/storage/cache from any additional DB/storage/cache provided by app developers.

Reply:
Yes, thank you, Ry! I am making the distinctions further down the page - so I thought it would be helpful to start off this way.